website/docs/r/cognito_identity_pool.markdown

```diff
@@ -70,14 +70,14 @@ backend and the Cognito service to communicate about the developer provider.
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - An identity pool ID in the format REGION:GUID.
+* `id` - An identity pool ID, e.g. `us-west-2_abc123`.
```

Comment:
Looking at the provider code where the ID is set ([here](https://github.com/hashicorp/terraform-provider-aws/blob/78b2dea023f65b5644f783663f32942227e16966/internal/service/cognitoidentity/pool.go#L155-L160)) and the AWS Go SDK for the [IdentityPool](https://docs.aws.amazon.com/sdk-for-go/api/service/cognitoidentity/#IdentityPool) type, I would expect this to be in the format of `REGION:GUID`, since that's what the Go SDK states the format will be in. Are you seeing a different output in practice?

Reply:
I tried again to confirm this:

```
$ terraform import aws_cognito_user_pool.pool1 us-west-2:3iqRdmz9p
╷
│ Error: error describing Cognito User Pool (us-west-2:3iqRdmz9p): InvalidParameterException: 1 validation error detected: Value 'us-west-2:3iqRdmz9p' at 'userPoolId' failed to satisfy constraint: Member must satisfy regular expression pattern: [\w-]+_[0-9a-zA-Z]+
│ 
│ 
╵

Releasing state lock. This may take a few moments...
$ terraform import aws_cognito_user_pool.pool1 us-west-2_3iqRdmz9p

Import successful!

The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.
```